app/models/project_media.rb

```diff
@@ -250,48 +250,51 @@ def replace_by(new_pm)
       raise I18n.t(:replace_blank_media_only)
     else
       # Save the new item
-      analysis = self.analysis
       new_pm.updated_at = Time.now
       new_pm.skip_check_ability = true
       new_pm.channel = { main: CheckChannels::ChannelCodes::FETCH }
       new_pm.save(validate: false) # To skip channel validation
-      new_pm.analysis = { title: analysis['title'], content: analysis['content'] }
-      # Point the claim
+
+      # Point the claim and consequently the fact-check
+      new_pm = ProjectMedia.find(new_pm.id)
       new_pm.claim_description = self.claim_description
```

Comment:
I think we can move this line before L256 and do one save instead of load item again and re-save it